src/components/buttons/Button/Button.jsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Icon } from "../../icons/Icon/Icon";
+
+import "./button.scss";
+
+/**
+ * Button
+ *
+ * All buttons used in the app
+ *
+ * @return {jsx}
+ */
+export const Button = ({
+  /* Add props here */
+  text,
+  iconName,
+  size,
+  color,
+  secondary,
+  ghost,
+  emergency,
+  disabled,
+  onClick,
+  ...rest
+}) => {
+  const mode = secondary
+    ? "btn--secondary"
+    : ghost
+    ? "btn--ghost"
+    : emergency
+    ? "btn--emergency"
+    : "btn--primary";
+
+  return (
+    <button
+      className={[
+        `btn btn--${size} btn--${color}`,
+        mode,
+        disabled && "btn--disabled",
+      ].join(" ")}
+      onClick={onClick}
+      {...rest}
+    >
+      {emergency ? (
```

Comment:
@georgiganchev-7DIGIT Let's not have a specific property for emergency as it can be simply built by passing the icon. I think iconName is enough.

Reply:
@vasilen-7DIGIT  I did it this way because the shape of the emergency button is different from the others